sunpy/coordinates/tests/test_frames.py

```diff
@@ -492,6 +493,21 @@ def test_heliocentric_radial_psi(x, y, psi):
 # ==============================================================================
 
 
+@pytest.mark.parametrize(('obstime', 'dipole_lonlat'),
+                         [('2012-07-01', [-72.408328, 80.16423]*u.deg),
```

Comment:
Do we need to test ther frames with astropy.times? Or are all the inputs converted internally?